### ISSUE TITLE
Temperature datafield shows 0.0

### ISSUE THREAD

**Issue by Herby** [fhorinek/SkyDrop#214]
I have noticed a small sw-bug on 2 different skydrops:
The temperature shows 0.0 and the correct temperature alternately with FW 3092. In FW 2650 it was ok.

Just a recommendation: It is not useful, to show a glideratio > 50. Just show "--" or "50+"
Thanks again for your good job.

**Comment by Herby** [comment]
Please can someone confirm, the 0.0 in the temperature field?
Thanks
Herby

**Comment by fitz07** [comment]
hello,
like you the widget shows 0.0 and the temperature alternately.
fitz

**Comment by fhorinek** [comment]
Dose the 0.0 appears after the correct temperature is displayed?

It takes around 2 seconds after start to read the temperature value (low priority data).

**Comment by fitz07** [comment]
I have the impression that the value "0.0" appears when the displayed temperature is not yet stabilisée.une both room temperature then reached the "0.0" disappeaes.actual temperature is not a priority but it's given a real bonus.it is just a hypothèse.i have sent you two emails from the address "jeando@caramail.com" you might also answer.
Thank you for your job.

**Comment by Herby** [comment]
Yes, the 0.0 appears some time after the correct temperature is displayed. With my 2 skydrops, old and new Version, it is easily reproducable.
Thanks for Fixing!
Herby

**Comment by Herby** [comment]
Bad News:
New Version 3122 still Shows 0.0 in the temperature field!
Herby

**Comment by Herby** [comment]
Still not working on Version 3248
Herby

**Comment by fhorinek** [comment]
@Herby 
should be fixed now, did you still suffer from this bug in 3395?

**Comment by Herby** [comment]
I will test it next flight!
Thanks
